### PR TITLE
Adding a github actions to automatically close stale issues/PRs based on CLOUDP-79100

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

This is a follow up of https://github.com/mongodb/terraform-provider-mongodbatlas/pull/843. This PR adds a cron job (https://github.com/marketplace/actions/close-stale-issues) that scans open issues/PRs and closes them based on the logic decided in [CLOUDP-79100: Autoclose stale issues and PRs](https://jira.mongodb.org/browse/CLOUDP-79100). We have recently enabled the same cron job for AtlasCLI ([#1490](https://github.com/mongodb/mongodb-atlas-cli/pull/1490)) and atlas go client ([#318](https://github.com/mongodb/go-client-mongodb-atlas/pull/318))

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: README.md
Comment (andreaangiolillo) [inline README.md:262]: keeping this consistent with the CLI and go client https://github.com/mongodb/mongodb-atlas-cli/pull/1490/files

**Comment by themantissa** [comment]
Let me clean up a few issues before we merge this in.  I should be able to do so before the end of the week.  Once I do I will approve.  @Zuhairahmed is this in line with the policy you wanted to implement.

**Review by Zuhairahmed — APPROVED** [review]
Yes, this is good github issue/PR hygiene and the 30/60 day clock is reasonable for now. we can always adjust cadence in future as needed. thanks again for automating this @andreaangiolillo! just FYI @martinstibbe

**Comment by themantissa** [comment]
@andreaangiolillo and @Zuhairahmed just to document it here. We are cleaning up a few last issues and then we'll institute this change.  Once we are ready @Zuhairahmed let me know and I'll approve to unblock.

**Review by themantissa — APPROVED** [review]
LGTM!

**Pull request merged**
